angular/projects/spark-angular/src/lib/components/sprk-card/sprk-card.stories.ts

```diff
@@ -124,24 +243,86 @@ export const highlightedHeader = () => ({
           signiferumque at. Numquam.
         </p>
       </div>
-  </sprk-card>`,
+    </sprk-card>
+  `,
 });
 
-highlightedHeader.story = {
-  name: 'Highlighted Header',
+legacyHighlightedHeader.story = {
+  name: 'Legacy Highlighted Header (Deprecated)',
   parameters: {
     docs: { iframeHeight: 300 },
     jest: ['sprk-card.component'],
   },
 };
 
 export const teaser = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-card idString="teaser">
+      <a
+        sprkLink
+        variant="unstyled"
+        href="#nogo"
+        sprkStackItem
```

Comment:
Should there be another `<sprk-stack>` in here wrapping all this?

Reply:
@guern1kn , no because the card component adds the Stack for us